src/GhSync/Options/IOptions.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace gh_sync;
+
+using Octokit;
+using Microsoft.VisualStudio.Services.WebApi;
+
+public interface IOptions
+{
+    string GetToken(string varName);
+
+    Task<Organization?> GetOrgProfile(IGitHubClient ghClient, string orgName);
```

Comment:
This is actually because of trying to mock out the functions required in `GetClient()` from GitHub. Because I can't mock other methods from GitHub if I'm running a test against GitHub, I had to move the `GetOrgProfile()` method to Options so that I could selectively mock different situations.